ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/TestCaseNode.java

```diff
@@ -264,12 +266,19 @@ private static Topic createTopicFromStatement(
       } else {
         avroSchema = Optional.empty();
       }
-      return new Topic(
+      return windowedSerdeFactory.map(wsf -> new Topic(
           topicName,
           avroSchema,
-          getSerdeSupplier(format),
+          wsf,
+          Utilities.getSerdeSupplier(format),
           KsqlConstants.legacyDefaultSinkPartitionCount,
-          KsqlConstants.legacyDefaultSinkReplicaCount);
+          KsqlConstants.legacyDefaultSinkReplicaCount,
+          Optional.empty())).orElseGet(() -> new Topic(
```

Comment:
nit: can we have the `orElseGet` on a new line? For a second I was really confused why this compiled haha 😂

Reply:
Changed it to if statement.